drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/widget/ExpressionBuilder.java

```diff
@@ -499,6 +500,12 @@ public void onClick( ClickEvent event ) {
                                 Window.alert( GuidedRuleEditorResources.CONSTANTS.TheVariableName0IsAlreadyTaken( var ) );
                                 return;
                             }
+                            String validationTextToDisplay = PatternBindingValidatorUtils.getValidationTextToDisplay(getModeller().getPatternBindingValidators(),
```

Comment:
And again [here](https://github.com/mcimbora/drools-wb/blob/e1b0183d6b588e3fd0c79da9bac0d20a43979a7f/drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/widget/ExpressionBuilder.java#L499).